packages/sandbox/package.json

```diff
@@ -3,9 +3,10 @@
   "private": true,
   "version": "0.0.0-development",
   "scripts": {
-    "test": "dotcom-tool-kit test:local",
     "build": "dotcom-tool-kit build:local",
-    "heroku-postbuild": "dotcom-tool-kit release:remote build:remote",
+    "format": "dotcom-tool-kit format:local",
+    "test": "dotcom-tool-kit test:local",
+    "heroku-postbuild": "dotcom-tool-kit cleanup:remote release:remote build:remote",
```

Comment:
```suggestion
    "heroku-postbuild": "dotcom-tool-kit release:remote build:remote cleanup:remote",
```

Reply:
As discussed in previous reviews, as `heroku-postbuild` already had `dotcom-tool-kit`, `cleanup:remote` was added in the wrong place.